src/App.tsx

```diff
@@ -1,49 +1,123 @@
-          <li>C</li>
-          <li>Dm</li>
-          <li>F</li>
-          <li>G</li>
-        </ol>
-      </div>
-      <div className="files">
-        <span>+ Files</span>
-          <div>Bass.mp3 <br></br> <br></br> 2 Comments</div>
-          <div>Drums.mp3 <br></br> <br></br> 2 Comments</div>
-          <div>Yodeling.mp3 <br></br> <br></br> 2 Comments  </div>
-          <div>Tuba.mp3 <br></br> <br></br> 2 Comments  </div>
-      </div>
-      <div className="comments">
-        <span>3 Comments</span>
-        <div className="display-comments">
-        <p><FontAwesomeIcon icon={faFaceSmile} /> Jerry: My name is Schmoopie and I love this song. It reminds me of my grandpa</p>
-        <p><FontAwesomeIcon icon={faFaceSmile} /> Larry: jokes on you! Im deaf.</p>
-        <p><FontAwesomeIcon icon={faFaceSmile} /> Terry: Notice the tinnitus ringing in your ears, let the hiss from your damaged hearing remind you of peaceful ocean waves...</p>
-        </div>
+
+
+
+  const sectionData = {
+    name: 'Intro',
+    description: 'This intro section consists of a tuba quartet in the style of DJ Templeton & The Windsurfers',
+    numRevisions: 42,
+  };
+
+  const currentChordProgression = ['C', 'Dm', 'F', 'G']
+
+  const files = [
+    {
+      title: 'Bass.mp3',
+      numComments: 2,
+    },
+    {
+      title: 'Drums.mp3',
+      numComments: 2,
+    },
+    {
+      title: 'Yodeling.mp3',
+      numComments: 2,
+    },
+    {
+      title: 'Tuba.mp3',
+      numComments: 2,
+    },
+  ]
+
+
+  const comments = [
+    {
+      name: 'Jerry',
+      commentText: 'My name is Schmoopie and I love this song. It reminds me of my grandpa'
+    },
+    {
+      name: 'Lerry',
+      commentText: 'jokes on you! Im deaf.'
+    },
+    {
+      name: 'Jerry',
+      commentText: 'Notice the tinnitus ringing in your ears, let the hiss from your damaged hearing remind you of peaceful ocean waves...'
+    },
+  ]
+
+
+  // Components
+
+  function Files() {
+    return <div className="files">
+      <span>+ Files</span>
+      {files.map((file) =>
+        <div>
+          {file.title}
+          <br></br> <br></br>
+          {file.numComments + ' '}
+          Comments
+        </div>)}
+    </div>
+  }
+
+
+  function ChordProgression() {
+    return <div className="chords">
+      <ol>
+        {currentChordProgression.map((chord, index) => <li>{chord}</li>)}
+      </ol>
+    </div>
+  }
+
+
+  function Comments() {
+    return <div className="comments">
+      <span>3 Comments</span>
+      <div className="display-comments">
+        {comments.map((comment, index) => {
+          return <>
+            <p><FontAwesomeIcon icon={faFaceSmile} /> {comments[index].name}: {comments[index].commentText}</p>
+          </>
+        })}
       </div>
-      <div className="submit">
+    </div>
+  }
+
+  function Submit() {
+    return <div className="submit">
       <textarea placeholder="make a comment"></textarea>
       <button>Submit</button>
+    </div>
```

Comment:
It probably makes sense to have the submit button trigger an update to the array of comments shown. That would make the code more ready for porting over.

We would add a prop to this `Submit` component for the parent component to process the form submission.

Also, I'm thinking this component can be called `CreateComment` to be more specific about what it's doing. We can then have the passed in callback prop to be called `onSubmit`. What do you think?

Reply:
I think that renaming the component to CreateComment is a good idea for sure, since the name is now more clear as to its purpose. 




I believe that in order for this component to update the array of comments shown, there are a few options we could go for, namely: 

- just adding a comment to the array (low complexity, but only a temporary soltion)
- adding the array to local storage and actually updating the array there (a bit more involved but more of a long term solution)

What are your thoughts on this?